### ISSUE TITLE
Docs: Create list of notcurses dependent applications

### ISSUE THREAD

**Issue by WSLUser** [dankamongmen/notcurses#2017]
It would be nice if there was a consolidated doc or something in the README that lists all the projects that use the notcurses library with links to the repos.

**Comment by dankamongmen** [comment]
i can only point to my own

* https://github.com/dankamongmen/growlight
* https://github.com/dankamongmen/omphalos

i know there are other projects using it, but i'm not sure which ones ought be pointed at, or where they are, or anything like that. there was definitely something in the debian archive that was using it, but i'm not seeing it now (besides growlight)? weird.

**Comment by WSLUser** [comment]
Yeah yours are easy enough to know.  I think it's based on metrics (how many downloads, stars, forks, etc. ) for other projects and typically are aware when people file bugs against the project. If you know of any applications that have filed bugs against you, it should be easy enough to list them. Seems like you catch most bugs yourself before anybody has a chance to file them, which is a good thing, though it certainly helps when you have somebody file instead so you can indeed know who is using your project and has a use-case that potentially triggered a bug.

I also noticed you have a Discussions tab, maybe you could create a new thread dedicated for determining this information and hopefully people who drop by here add projects to the list.

**Comment by dankamongmen** [comment]
does anybody know projects that are using it? ought i create some list that people can add to?

i don't want to start with an empty list =]
